image-loader/app/model/Uploader.scala

```diff
@@ -77,6 +84,43 @@ case class ImageUploadOpsDependencies(
 
 object Uploader extends GridLogging {
 
+def toQuarantineUploadOpsCfg(
+      config: ImageLoaderConfig): ScanImageUploadOpsCfg = {
+    ScanImageUploadOpsCfg(config.quarantineBucket)
+  }
+
+  def fromQuarantineUploadRequestShared(
+      uploadRequest: UploadRequest,
+      deps: QuarantineImageUploadOpsDependencies)(
+      implicit ec: ExecutionContext,
+      logMarker: LogMarker): Future[JsObject] = {
+
+    import deps._
+
+    uploadAndStoreQuarantineImage(config,
+                                  storeOriginalFile,
+                                  uploadRequest,
+                                  deps)
```

Comment:
Both the deps object and individual fields from it are passed through into this function. It would be cleaner if one approach was chosen.